specific/cmo_project/models/project.py

```diff
@@ -293,15 +315,116 @@ def _get_state_before_inactive(self):
     @api.multi
     def _set_project_analytic_account(self):
         for project in self:
-            parent_project = self.env['project.project'].browse(project.project_parent_id.id)
+            parent_project = self.env['project.project'].browse(
+                project.project_parent_id.id)
             project.parent_id = parent_project.analytic_account_id.id
 
     @api.multi
     @api.constrains('brief_date', 'date')
     def _check_brief_dates(self):
         self.ensure_one()
         if self.brief_date > self.date:
-            return ValidationError("project brief-date must be lower than project end-date.")
+            return ValidationError("project brief-date must be lower than \
```

Comment:
For long message, python can write like this.
```
return ValidationError("project brief-date must be lower than"
                                   "project end-date.")
```
Not hat we can new line without \ or +